deployment/dockerfile.appserver

```
@@ -49,3 +49,4 @@ USER www-data
 COPY --chown=www-data:www-data entrypoint.d/* /entrypoint.d/
 
 COPY --chown=www-data:www-data appserver_config/ /opt/iqgeo/config/
+# RUN --mount=type=secret,id=oidc,target=/opt/iqgeo/config/oidc/conf.json
```

```suggestion
RUN --mount=type=secret,id=oidc,target=/opt/iqgeo/config/oidc/conf.json
```

This will need some re-touching if it is to account for all _'appserver_config'_ scenarios, but does illustrate [exposing secrets to the build](https://docs.docker.com/reference/cli/docker/buildx/build/#secret) as a best practice.

---

Do you have to add this to the tools image as well?

---

@LucasThelen Negative. At least, not in current implementation. It does not `COPY` anything from the host file system. 

Now, that said, if tools image **does need** the `/opt/iqgeo/config/` directory (meaning, it does use the OIDC configuration _currently_), its unclear to me how that directory/path is getting into that image now, because it is **not** explicitly doing so in the [`dockerfile.tools`](deployment/dockerfile.tools). 
Obviously, if that _should be_ a requirement, yes, it can be done there too.